specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json

```diff
@@ -0,0 +1,1360 @@
+{
+  "consumes": [
+    "application/json"
+  ],
+  "definitions": {
+    "ErrorResponse": {
```

Comment:
What's the motivation to use this custom type instead of reusing the one from common-type?